test/ret/node_stat_test.exs

```diff
@@ -0,0 +1,100 @@
+defmodule Ret.NodeStatTest do
+  use Ret.DataCase
+  alias Ret.{NodeStat, Repo}
+
+  describe "NodeStat.max_ccu_for_time_range() tests" do
+    @tag marked: true
```

Comment:
Remember to remove the tags. Same below.
(We should probably add a lint rule for this if we can some time, I forget to do it too)